components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/src/main/java/org/wso2/carbon/identity/oauth2/grant/organizationswitch/OrganizationSwitchGrant.java

```diff
@@ -56,6 +61,7 @@ public class OrganizationSwitchGrant extends AbstractAuthorizationGrantHandler {
 
     private static final Log LOG = LogFactory.getLog(OrganizationSwitchGrant.class);
     private static final String TOKEN_BINDING_REFERENCE = "tokenBindingReference";
+    private static final String OAUTH_APP_PROPERTY = "OAuthAppDO";
```

Comment:
As a future improvement, lets use a constant define in the OAuth repo
ex- https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/AuthorizationHandlerManager.java#L69